gazebo/gui/ModelMaker.cc

```diff
@@ -116,6 +118,14 @@ bool ModelMaker::InitFromFile(const std::string &_filename)
     }
     return false;
   }
+
+  // Since the above didn't fail, we assume the file exists
+  // We also assume that _filename is a path to a file, not to a model directory
```

Comment:
is this assumption safe enough or should we add a check for a file?

Reply:
As far as I can tell, all the code paths that call `InitFromFile` have a file as an argument. 
* From tests in https://github.com/osrf/gazebo/blob/gazebo11/gazebo/gui/ModelMaker_TEST.cc
* From InsertModelWidget via a `createEntity` event https://github.com/osrf/gazebo/blob/7200818c3f1b905505d25c66d5ad004cd0003e88/gazebo/gui/InsertModelWidget.cc#L319-L326